packages/crew_preload.rb

```diff
@@ -0,0 +1,47 @@
+# This package creates a /usr/local/lib64 directory on armv7l/aarch64
+# systems to handle the aarch64 crew-preload.so being installed to
+# handle the default system aarch64 binaries now present on recent
+# multiarch aarch64/armv7l based ARM ChromeOS milestones.
+# These binaries are built as per the instructions at
+# https://github.com/chromebrew/crew-package-glibc/tree/main/crew-preload
+require 'package'
+
+class Crew_preload < Package
+  description 'Chromebrew LD_PRELOAD library to allow a mixed glibc environment.'
+  homepage 'https://github.com/chromebrew/crew-package-glibc'
+  version '2.41-5'
+  license 'GPL-3'
+  compatibility 'all'
+  source_url 'SKIP'
+  binary_compression 'tar.zst'
+
+  binary_sha256({
+    aarch64: 'cbfad316ea4bc536fef57f0f104e05baca87288b917b53cc84f1614c786bf134',
+     armv7l: 'cbfad316ea4bc536fef57f0f104e05baca87288b917b53cc84f1614c786bf134',
+       i686: 'b19f235f07480a309d06ebd0107a9cb510ab01ac5ac975c734be62ab156d9da1',
+     x86_64: 'ff4fb4407dd4337b74606272cab7f1e8cdafa0c99e9bb7e07485c503bbbac267'
+  })
+
+  conflicts_ok
+  no_fhs if %w[aarch64 armv7l].include?(ARCH)
```

Comment:
Not needed if putting libraries in the correct directories?

Reply:
Ah! We made crew handle this already!